study.md

```diff
@@ -27,17 +27,20 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+It's not the HTML you write and it's not the view source either.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+They are in some way. Dev Tools is similar to HTML when opening
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.